### PR TITLE
Modified comment block

### PR DESCRIPTION
**Pull request by medz**

### CODE REVIEW

File: src/Composer/Util/RemoteFilesystem.php [hunk 1]
```diff
@@ -114,13 +114,21 @@ public function getOptions()
...
      *
-     * @return array $options
+     * @param array $options
+     * @return self
      */
     public function setOptions(array $options)
     {
         $this->options = array_replace_recursive($this->options, $options);
+        
+        return $this;
```
[the remark this answers is not in the document]
Comment (medz) [reply src/Composer/Util/RemoteFilesystem.php]: > do you actually have a use-case for this build pattern here?

No effect on building this return, I think this helps with chained calls

[the remark this answers is not in the document]
Comment (medz) [reply src/Composer/Util/RemoteFilesystem.php]: @staabm Wait a moment, I add a test case

[the remark this answers is not in the document]
Comment (medz) [reply src/Composer/Util/RemoteFilesystem.php]: @staabm I checked all the code and only used this method in one place, so I removed the chained mode code!

https://github.com/composer/composer/blob/master/src/Composer/Repository/ComposerRepository.php#L105

**Comment by Seldaek** [comment]
Thanks

**Pull request merged**